src/containers/router.js

```diff
@@ -34,6 +34,7 @@ export default class Root extends Component {
 
   componentWillMount () {
     let currentPath = store.getState().routing.locationBeforeTransitions.pathname;
+    window.history.replaceState('', '', currentPath);
```

Comment:
I guess it possibly just needs some kind of whitelist filter, so that it only removes parameters that we don't specify that we need.